src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -5,13 +5,46 @@
 import com.google.inject.Inject;
```

Comment:
This file somehow escaped the reformat and I'm not sure how. I removed the `extractFilesFromMultiPart` method and moved it to a static method in `Resource.java`.